views/base.html

```diff
@@ -55,6 +55,9 @@
     </div> -->
 
     <main>
+      {% block communities %}
```

Comment:
As it stands, it includes the communities selector. It wouldn't be used anywhere else.